apps/frontend/src/network/Routes.tsx

```diff
@@ -19,12 +20,23 @@ const Network: React.FC<{}> = () => {
   const { search } = useLocation();
   const history = useHistory();
   const currentUrlParams = new URLSearchParams(search);
-
+  const filters = new Set(currentUrlParams.getAll('filter'));
+  const searchQuery = currentUrlParams.get('searchQuery') || undefined;
+  const [searchQueryDebounce] = useDebounce(searchQuery, 400);
```

Comment:
```suggestion
  const [searchQueryDebounce] = useDebounce(searchQuery, 500);
```
lol @ two different numbers :D